package.json

```diff
@@ -39,6 +39,7 @@
   "devDependencies": {
     "gh-pages": "~0.3.1",
     "moment": "~2.3.0",
+    "shallow-settings": "^0.1.0",
```

Comment:
Please use `~` for semver fuzziness for both consistency and less likelihood of things breaking